app/src/main/java/org/openimis/imispolicies/Enquire.java

```diff
@@ -393,8 +393,8 @@ public void run() {
                              TotalDelivieriesLeft = jsonObject.getString("totalDelivieriesLeft").equalsIgnoreCase("null")? "": "TotalDelivieriesLeft: "+jsonObject.getString("totalDelivieriesLeft");
                              TotalAntenatalLeft = jsonObject.getString("totalAntenatalLeft").equalsIgnoreCase("null")? "": "TotalAntenatalLeft: "+jsonObject.getString("totalAntenatalLeft");
                              ConsultationAmountLeft = jsonObject.getString("consultationAmountLeft").equalsIgnoreCase("null")? "": "ConsultationAmountLeft: "+jsonObject.getString("consultationAmountLeft");
-                             SurgeryAmountLeft = jsonObject.getString("surgeryAmountLeft").equalsIgnoreCase("null")? "": "TotalAdmissionsLeft: "+jsonObject.getString("surgeryAmountLeft");
-                             HospitalizationAmountLeft = jsonObject.getString("hospitalizationAmountLeft").equalsIgnoreCase("null")? "": "TotalAdmissionsLeft: "+jsonObject.getString("hospitalizationAmountLeft");
+                             SurgeryAmountLeft = jsonObject.getString("surgeryAmountLeft").equalsIgnoreCase("null")? "": "SurgeryAmountLeft: "+jsonObject.getString("surgeryAmountLeft");
```

Comment:
"SurgeryAmountLeft: " is hardcoded. Please add this in strings.xml.

Reply:
Fixed